_data/settings/components/identifier.yml

```diff
@@ -0,0 +1,27 @@
+name: Identifier
+contents:
+  - name: Background color
+    description: The background color of the identifier. Use a color of **grade 80 or higher**, darker than the section that precedes it.
+    var: $theme-identifier-background-color
+    default: '"base-darkest"'
+    type: color
+  - name: Font family
+    description: The font family of the identifier
+    var: $theme-identifier-font-family
+    default: '"ui"'
+    type: family
+  - name: Domain text color
+    description: The color of your domain text in the identifier masthead. This should be **grade 20-30** in the same family as the `$theme-identifier-background-color`.
+    var: $theme-identifier-identity-domain-color
+    default: default
```

Comment:
Should this be `"primary-lighter"`?

Reply:
No — we intend it to be the same family as the component background, but I see that's not how it's set up in settings. Changing... Good catch